src/Control/Inciter/Types.h

```diff
@@ -104,6 +104,18 @@ using CompFlowPDEParameters = tk::tuple::tagged_tuple<
   //! Parameter vector (for specific, e.g., verification, problems)
   tag::beta, std::vector< kw::pde_beta::info::expect::type >,
   //! Parameter vector (for specific, e.g., verification, problems)
+  tag::betax, std::vector< kw::pde_betax::info::expect::type >,
+  //! Parameter vector (for specific, e.g., verification, problems)
```

Comment:
Are all of these parameter vectors for verification problems? Please provide more context